### DISCUSSION TITLE
Can't Verify School Email

### DISCUSSION THREAD

**Discussion by binomial1** [community/community#102567] [GitHub Education]
### Hi everyone,

Hello! I have been trying to apply to GitHub Global Campus, but I can't verify my school email address. My school district, after a cyberattack, set it so that school emails are restricted from sending or receiving emails from anyone who does not have an email associated with my school district. How would I be able to complete my application process?

**Comment by onurravli** [comment]
I am sorry to say that but I don't think there are no solutions on the GitHub side. Did you talk to your school's IT dept? I think they can help.
